app/services/group_service.rb

```diff
@@ -17,8 +17,9 @@ def self.create(group:, actor:)
 
   def self.update(group:, params:, actor:)
     actor.ability.authorize! :update, group
-    group.update params
-    group
+    group.assign_attributes(params)
+    group.group_privacy = params[:group_privacy] if params[:group_privacy]
+    group.tap(&:save)
```

Comment:
I've been meaning to write this for a while.. It's my definition that I've been writing services to meet.

Want to get your impression/input of it

https://github.com/loomio/tech-manual/blob/master/ruby_services.md

One thing I think we differ on is the return value... care to share your thoughts?